backend/src/main/java/com/fake_orgasm/generator/flight_history_generator/Airport.java

```diff
@@ -0,0 +1,74 @@
+package com.fake_orgasm.generator.flight_history_generator;
+
+public class Airport {
+    private String airportName;
+    private String country;
+    private String state;
+
+    /**
+     * This is the constructor for this class.
+     *
+     * @param airportName The name of the airport.
+     * @param country     The country where the airport is located.
+     * @param state       The state where the airport is located.
+     */
+    public Airport(String airportName, String country, String state) {
+        this.airportName = airportName;
+        this.country = country;
+        this.state = state;
+    }
+
+    /**
+     * This method returns the name of the airport.
+     *
+     * @return The airport's name.
+     */
+    public String getAirportName() {
```

Comment:
We use Lombok for the generation of getters and setters, could you implement the notation for the getters and setters of lombok instead of implementing them manually

Reply:
The suggestion has been applied.